sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/feedranges/FeedRangeInternalSerializer.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.feedranges;
+
+import com.azure.cosmos.implementation.Constants;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+
+public class FeedRangeInternalSerializer extends StdSerializer<FeedRangeInternal> {
```

Comment:
This shouldn't be neceesary. FeedRange has publi toString/fromString (both returning or accepting json representation of teh feed range) - you can simply use that instead of creating competing way to serialize/deserialize